src/components/Footer/Footer.jsx

```diff
@@ -4,13 +4,17 @@ import styles from './Footer.scss';
 function Footer() {
   const [date, setDate] = useState('');
 
-  const timerID = setInterval(() => setDate(new Date()), 1000);
+  useEffect(() => {
+    const timerId = setInterval(() => setDate(new Date()), 1000);
 
-  useEffect(() => {}, [timerID]);
+    return () => {
+      clearTimeout(timerId);
+    };
+  }, [date]);
```

Comment:
In this case you don't have to specify date in a list of dependencies. Try it and make sure you understand why exactly